config/site_settings.yml

```diff
@@ -1919,7 +1919,7 @@ user_api:
   allow_user_api_keys:
     default: true
   allow_user_api_key_scopes:
-    default: "read|write|message_bus|push|notifications|session_info"
+    default: "read|write|message_bus|push|notifications|session_info|one_time_password"
```

Comment:
what is the transition period going to look like now that existing keys kind of want to have one_time_password?

Reply:
It's pretty straightforward: sites without the one_time_password scope won't get the redirect treatment. This is valid for both sites that aren't up to date as well as sites that decide to disallow that scope. 

The PR also bumps the user API version to 4, so apps can check whether `one_time_password` is supported by checking the API version.